tests/tracing-tests/test-trace.ts

```diff
@@ -509,3 +509,119 @@ describeDevMoonbeam("Trace", (context) => {
     });
   });
 });
+
+describeDevMoonbeam("Trace", (context) => {
+  it("should correctly trace subcall (call list)", async function () {
+    this.timeout(10000);
+
+    const { contract: contractProxy, rawTx } = await createContract(context, "TestCallList");
+    await context.createBlock({ transactions: [rawTx] });
+
+    const { contract: contractDummy, rawTx: rawTx2 } = await createContract(
+      context,
+      "TestContract"
+    );
+    await context.createBlock({ transactions: [rawTx2] });
+
+    // console.log(`Proxy : ${contractProxy.options.address}`);
+    // console.log(`Dummy : ${contractDummy.options.address}`);
+
+    let callTx = await context.web3.eth.accounts.signTransaction(
+      {
+        from: GENESIS_ACCOUNT,
+        to: contractProxy.options.address,
+        gas: "0x100000",
+        value: "0x00",
+        data:
+          // selector for call
+          "0x1b8b921d" +
+          // address of subcall contract
+          contractDummy.options.address.slice(2).padStart(64, "0") +
```

Comment:
Consider using ethers.utils.Interface (https://github.com/PureStake/moonbeam/blob/f67d0188cfee414d8dee90d959c86b8ff7a9b40b/tests/tests/test-precompile/test-precompile-xtokens.ts#L22) so that its easier to follow how the input ins constructed